Sources/Rendering/Core/Picker/index.d.ts

```diff
@@ -127,6 +118,11 @@ export function newInstance(initialValues?: IPickerInitialValues): vtkPicker;
  * the prop itself, the data set, and the mapper. 
  * For vtkPicker the closest prop is the one whose center point (i.e., center of bounding box) projected on the view ray is closest
  * to the camera. Subclasses of vtkPicker use other methods for computing the pick point.
+ * 
+ * vtkPicker is used for quick geometric picking. If you desire more precise
```

Comment:
`If you desire` -> use passive tense (.e.g More precise picking... must use the subclasses...)